server/src/main/java/com/flipkart/varadhi/web/v1/TopicHandlers.java

```diff
@@ -49,15 +50,18 @@ public List<RouteDefinition> get() {
                 "/v1/tenants/:tenant/topics",
                 List.of(
                         new RouteDefinition(
-                                HttpMethod.GET, "/:topic", Set.of(), this::get,
+                                HttpMethod.GET, "/:topic", Set.of(), Sets.newLinkedHashSet(Sets.newHashSet()),
```

Comment:
new LinkedHashSet() is recommended directly ? Also is initialise with empty set needed ?